src/index.ts

```diff
@@ -3,7 +3,9 @@ import { Options } from "./types";
 
 export { memoryStore } from "./stores/memory.store";
 export { redisStore } from "./stores/redis.store";
-export { GetCacheKey, PurgeFunction } from "./types";
+export { GetCacheKey, GetCacheTag, PurgeFunction } from "./types";
 
 export type ExpressAggressiveCacheOptions = Options;
+
+export { akamaiGetCacheTag } from "./cache-tag";
```

Comment:
I think this should live in the README as an example on how to use the `getCacheTag` function.